GILua/src/main/java/org/anime_game_servers/gi_lua/script_lib/ScriptLib.kt

```diff
@@ -3728,6 +3728,1057 @@ object ScriptLib {
+                )
+            }
+        }
+    }
+
+    @JvmStatic
+    fun BeginCameraSceneLookWithTemplate(context: LuaContextWrapper, var1: Int, camParam: LuaTable): Int {
+        return context.onGroupContext {
+            onScriptLibHandler {
+                beginCameraSceneLookWithTemplate(this@onGroupContext, var1, camParam)
+            }
+        }
+    }
+
+    @JvmStatic
+    fun CheckIsInGroup(context: LuaContextWrapper, groupId: Int, configId: Int): Boolean {
+        return context.onGroupContext {
+            onScriptLibHandler {
+                checkGroupIdAndConfigId(::CheckIsInGroup, groupId, configId)?.let {
+                    return@onScriptLibHandler it.getValue() < 0
+                }
+                checkIsInGroup(this@onGroupContext, groupId, configId)
+            }
+        }
+    }
+
+    @JvmStatic
+    fun ClearExhibitionReplaceableData(context: LuaContextWrapper, uid: Int, key: String): Int {
+        return context.onGroupContext {
+            onScriptLibHandler {
+                clearExhibitionReplaceableData(this@onGroupContext, uid, key)
+            }
+        }
+    }
+
+    @JvmStatic
+    fun ClearPoolMonsterTide(context: LuaContextWrapper, groupId: Int, tideNum: Int): Int {
+        return context.onGroupContext {
+            onScriptLibHandler {
+                checkGroupId(::ClearPoolMonsterTide, groupId)?.let {
+                    return@onScriptLibHandler it.getValue()
+                }
+                clearPoolMonsterTide(this@onGroupContext, groupId, tideNum)
+            }
+        }
+    }
+
+    @JvmStatic
+    fun ContinueAutoMonster(context: LuaContextWrapper, groupId: Int, tideNum: Int): Int {
+        return context.onGroupContext {
+            onScriptLibHandler {
+                checkGroupId(::ContinueAutoMonster, groupId)?.let {
+                    return@onScriptLibHandler it.getValue()
+                }
+                continueAutoMonster(this@onGroupContext, groupId, tideNum)
+            }
+        }
+    }
+
+    @JvmStatic
+    fun ContinueTimeAxis(context: LuaContextWrapper, key: String): Int {
+        return context.onGroupContext {
+            onScriptLibHandler {
+                continueTimeAxis(this@onGroupContext, key)
+            }
+        }
+    }
+
+    @JvmStatic
+    fun CreateFoundation(
+        context: LuaContextWrapper,
+        uidList: IntArray,
+        configId: Int,
+        groupId: Int,
+        index: Int
+    ): Int {
+        return context.onGroupContext {
+            onScriptLibHandler {
+                checkGroupIdAndConfigId(::CreateFoundation, groupId, configId)?.let {
+                    return@onScriptLibHandler it.getValue()
+                }
+                createFoundation(this@onGroupContext, uidList, configId, groupId, index)
+            }
+        }
+    }
+
+    @JvmStatic
+    fun CreateFoundations(context: LuaContextWrapper, fundationTable: LuaTable, groupId: Int, index: Int): Int {
```

Comment:
Scriptlib doesn't directly receive an object implementing the LuaTable interface, you need have the parameter as type `Any` (similar to Object in java), and then the the LuaTable object via `context.engine.getTable(rawTable)`.
Check for example EnterPersistentDungeon for one example.